crates/cli/src/init.rs

```diff
@@ -473,6 +510,43 @@ pub fn generate_grammar_files(
                         contents = contents.replace(r#"    c_config.flag("-utf-8");"#, &indented_replacement);
                     }
 
+                    // Introduce configuration variables for dynamic query inclusion
```

Comment:
Another check here.

And while you're at it, move the above replacement inside the wasm32 check.